### PR TITLE
Implemented sorting for zend-lucene-adapter

### PR DESCRIPTION
**Pull request by wachterjohannes**
This feature introduces simple sorting for zend-lucene adapter.

Additionally this PR enables empty searches for elastic-search and zend-lucene.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ CHANGELOG
 =========
 
 * dev-develop
+    * FEATURE     #97 Implemented sorting for zend-lucene-adapter
+    * FEATURE     #79 Consider limit + offset for searching
```
Comment (wachterjohannes) [inline CHANGELOG.md]: this entry was forgotten.

**Comment by danrot** [comment]
Is that still needed? Or have you decided to go another way? /cc @chirimoya

**Comment by danrot** [comment]
Based on the discussion we once had with @chirimoya I would still say that sorting is not part of the scope of this bundle. Therefore I would even go the other way and remove the sort functionality completely (Although it is a BC break, but we haven't released 1.0 anyway, so now is the chance to get rid of this)

**Comment by wachterjohannes** [comment]
lets discuss this tomorrow with @chirimoya

**Pull request closed without merging**